readthedocs.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
Could you please add a target in the [Makefile](https://github.com/Ericsson/codechecker/blob/master/Makefile) which will build the documentation files (`build_mkdocs`)?

Reply:
Yes but I would put that into a separate pull request because the two things are different.

Reply:
See  #2026